resources/assets/js/translations/locales/fr.json

```diff
@@ -2,18 +2,28 @@
   "activity.commentLocation.label": "Commentaire trouvé",
   "activity.commentMetadata": "{name} ({userRole}) a commenté à {time}.",
   "activity.viewComment.label": "Visualiser le commentaire",
+  "activityfeed.accordionAccessibleLabel": "Cliquez pour voir...",
   "activityfeed.error": "Une erreur s'est produite",
+  "activityfeed.header": "Cliquez pour voir les commentaires {totalActivities}",
   "activityfeed.loading": "Chargement de vos activités...",
-  "activityfeed.locations.preview": "Afficher l’aperçu de la page",
-  "activityfeed.locations.review.basicInfo": "Constructeur d'Affiche > Renseignements de base",
-  "activityfeed.locations.review.details": "Constructeur d'Affiche > Général",
-  "activityfeed.locations.review.environment": "Constructeur d'Affiche > Environnement de travail",
-  "activityfeed.locations.review.heading": "JConstructeur d'Affiche > Titre de la page de l’emploi",
-  "activityfeed.locations.review.impact": "Constructeur d'Affiches > Impact",
-  "activityfeed.locations.review.langRequirements": "Constructeur d'Affiche > Exigences linguistiques",
-  "activityfeed.locations.review.skills": "Constructeur d'Affiches > Compétences",
-  "activityfeed.locations.review.tasks": "Constructeur d'Affiches > Taches",
-  "activityfeed.locations.summary": "Résumé de l'emploi",
+  "activityfeed.loadingIconText": "Nombre d'activités est en cours de chargement...",
+  "activityfeed.locations.applicantReview.generic": "Général",
+  "activityfeed.locations.applicantReview.notUnderConsideration": "No Longer Under Consideration",
+  "activityfeed.locations.applicantReview.optionalConsideration": "Optional Consideration",
+  "activityfeed.locations.applicantReview.underConsideration": "Under Consideration",
```

Comment:
Should get these localized

Reply:
I'm don't think we have translations for these right now, but I yes they should be. I'll check with Gray if the deepl.com translations will suffice.

Reply:
Not having translations led me to #2882 which I think has come up a few times. Might want to tackle that first so we can get the related text translated as well.

Reply:
Ya forsure, I can do that. Also, looks like there might be a change to the English copy, but will find out tomorrow.